services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -648,6 +660,31 @@ export const RateDetails = ({
                                                                 }
                                                             />
                                                         </FormGroup>
+                                                        {ratesAcrossSubmissions && (
+                                                            <FormGroup>
+                                                                <Checkbox
+                                                                    id={`sharedRateCheckBox-${index}`}
```

Comment:
Didn't we recently add an `id` to rateInfo?  We could use that here.

Reply:
We do! But not when adding a new rate certification since `id` is generate in `toProtoBuffer.ts`. I'm already generating a `uuid` `key` in rateInfo, so I'll use that!